### ISSUE TITLE
Make edit profile accessible by dropdown menu in All CASA Admin dashboard

### ISSUE THREAD

**Issue by Zrrrpy** [rubyforgood/casa#631]
Part of epic #527

**What type of user is this for? [volunteer/admin/supervisor/all or All CASA Admin]**
All CASA Admin

**Where does/should this occur?**
In the All CASA Admin dashboard (dependent on #628)

**Description**
All CASA Admin needs an entry point to their Edit Profile page. It should be accessible by clicking on their email in the upper right hand corner of their user dashboard. When they click, "Edit Profile" should appear as the only option in a drop down menu. 

Log in as any other user to see how this should look. <a href="https://github.com/rubyforgood/casa/blob/master/db/seeds.rb>Login instructions here.</a>

**Screenshots**
It should look like this:
![Screen Shot 2020-08-31 at 6 30 51 PM (2)](https://user-images.githubusercontent.com/62810851/91784656-45749700-ebb8-11ea-8dde-50e91d5f3016.png)

Relates to issues: #627, #628, #630

**Comment by jmkoni** [comment]
Don't know who did this, but this is already done
<img width="577" alt="Screen Shot 2020-09-13 at 10 30 42 AM" src="https://user-images.githubusercontent.com/1082370/93020645-30323c00-f5ac-11ea-9fb9-9b8ec03da70f.png">
